integration/query_test.go

```diff
@@ -950,7 +951,19 @@ func TestQueryBatchSize(t *testing.T) {
 
 func TestQueryCommandGetMore(t *testing.T) {
 	t.Parallel()
-	ctx, collection := setup.Setup(t)
+
+	q := url.Values{}
+
+	// set 1 to ensure only one pool exists duration of the test,
+	// which forces a client to use a single connection pool
+	q.Set("maxPoolSize", "1")
+	q.Set("minPoolSize", "1")
+
+	s := setup.SetupWithOpts(t, &setup.SetupOpts{
+		ExtraOptions: q,
```

Comment:
```suggestion
		ExtraOptions: url.Values{
			"minPoolSize": []string{"1"},
			"maxPoolSize": []string{"1"},
		}
```

or something like that